exporter/otlpexporter/otlp.go

```diff
@@ -167,15 +167,15 @@ func processError(err error) error {
 	}
 
 	// Now, this is a real error.
-	retryInfo := getRetryInfo(st)
+	retryInfo := statusutil.GetRetryInfo(st)
 
 	if !shouldRetry(st.Code(), retryInfo) {
 		// It is not a retryable error, we should not retry.
 		return consumererror.NewPermanent(err)
 	}
 
 	// Check if server returned throttling information.
-	throttleDuration := getThrottleDuration(retryInfo)
+	throttleDuration := retryInfo.GetRetryDelay().AsDuration()
 	if throttleDuration != 0 {
```

Comment:
Will fix in a followup to always include the throttle information if present and not retryable.